exporters/prometheus/config.go

```diff
@@ -132,7 +132,7 @@ func WithoutScopeInfo() Option {
 // have special behavior based on their name.
 func WithNamespace(ns string) Option {
 	return optionFunc(func(cfg config) config {
-		if model.NameValidationScheme != model.UTF8Validation {
+		if model.NameValidationScheme != model.UTF8Validation { // nolint:staticcheck // we need this check to keep supporting the legacy scheme
```

Comment:
nit comment - applies to other places as well

```suggestion
		if model.NameValidationScheme != model.UTF8Validation { // nolint:staticcheck // We need this check to keep supporting the legacy scheme.
```